### PR TITLE
Fixes #5780 - focus on PSReadline

### LINKED ISSUE #5780: Reframe the topic to focus on the default behavior with respect to keyboard shortcuts
I suggest reversing the framing of the topic:

It is `PSReadLine` that controls keyboard shortcuts, including history-related ones.

In the major hosts  - the `ConsoleHost` for running in console windows / terminals, as well as the PowerShell Integrated Console in Visual Studio - `PSReadLine` is _loaded by default_, so that will be the default experience for most users.

**While it's possible to _opt-out_ by _manually unloading_ `PSReadLine` _focusing_ on that opt-out experience, the way the topic currently does,  is confusing** - not least because it is limited to the behavior of `conhost.exe` (Windows console windows).

Therefore:

* Stress the default behavior (`PSReadLine`).

  * Listing _`PSReadLine`'s_ history-related default key bindings would be helpful (in addition to the existing link to the `PSReadLine` documentation).

* Clearly state that the key bindings currently listed:

   *  (a) apply to _`conhost.exe`_ windows on _Windows only_ with _`PSReadLine` manually unloaded_
   * and that (b) it is then generally _up to whatever host program is running PowerShell_ to provide history-related keyboard shortcuts, _if any_ (Windows Terminal and the Unix terminals don't offer any, from what I can tell).

Honestly, given that there's little reason _not_ to use `PSReadLine`, **I'm not sure that documenting the `conhost.exe` keyboard shortcuts is even necessary** - a link to the `conhost.exe` documentation would suffice, and perhaps a note to `cmd.exe` user that the keybindings they're used to won't work (but could be configured to work similarly via `PSReadLine`).

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 1777470c-18a0-e5e6-2a51-7f432d8e18fd
* Version Independent ID: 936ffb0f-739d-e58c-360e-e05a9129b186
* Content: [about_History - PowerShell](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_history?view=powershell-7)
* Content Source: [reference/7.0/Microsoft.PowerShell.Core/About/about_History.md](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/reference/7.0/Microsoft.PowerShell.Core/About/about_History.md)
* Product: **powershell**
* GitHub Login: @SteveL-MSFT
* Microsoft Alias: **slee**

### PR DESCRIPTION
**Pull request by sdwheeler**
# PR Summary

Fixes [AB#1720127](https://dev.azure.com/mseng/677da0fb-b067-4f77-b89b-f32c12bb8617/_workitems/edit/1720127) - Fixes #5780 - focus on PSReadline

## PR Context

Select the type(s) of documents being changed.

**Cmdlet reference & about_ topics**
- [x] Version 7.1 preview content
- [x] Version 7.0 content
- [x] Version 6 content
- [x] Version 5.1 content

**Conceptual content**
- [ ] Fundamental conceptual articles
- [ ] Script sample articles
- [ ] DSC articles
- [ ] Gallery articles
- [ ] JEA articles
- [ ] WMF articles
- [ ] SDK articles
- [ ] Community content

## PR Checklist

- [x] I have read the [contributors guide][contrib] and followed the style and process guidelines
- [x] PR has a meaningful title
- [x] PR is targeted at the _staging_ branch
- [x] All relevant versions updated
- [x] Includes content related to issues and PRs - see [Closing issues using keywords][key].
- [x] This PR is ready to merge and is not **Work in Progress**
  - If the PR is work in progress, please add the prefix `WIP:` or `[WIP]` to the beginning of the
    title and remove the prefix when the PR is ready.

[contrib]: https://docs.microsoft.com/powershell/scripting/community/contributing/overview
[key]: https://help.github.com/en/articles/closing-issues-using-keywords

### CODE REVIEW

File: reference/5.1/Microsoft.PowerShell.Core/Add-History.md [hunk 1]
```diff
@@ -21,80 +21,109 @@ Add-History [[-InputObject] <PSObject[]>] [-Passthru] [<CommonParameters>]
...
-You can use this cmdlet to add specific commands to the history or to create a single history file that includes commands from more than one session.
+The `Add-History` cmdlet adds entries to the end of the session history, that is, the list of
+commands entered during the current session.
+
+You can use the `Get-History` cmdlet to get the commands and pass them to `Add-History`, or you can
+export the commands to a CSV or XML file, then import the commands, and pass the imported file to
+`Add-History`. You can use this cmdlet to add specific commands to the history or to create a single
+history file that includes commands from more than one session.
+
+The session history is a list of the commands entered during the session. The session history
```
Comment (chasewilson) [inline reference/5.1/Microsoft.PowerShell.Core/Add-History.md:33]: I would move this section above the one above it. It flows better informationally to me. 
**Current:**
- Brief overview of the command itself.
- Informs the user of a way to keep history outside of just the current session with current cmdlet.
- Fills in the user on the necessary information about session histories and how they are built.
- Informs user about **PSReadLine**. 

**Suggested:**
- Brief overview of the command itself.
- Fills in the user on the necessary information about session histories and how they are built.
- Informs the user of a way to keep history outside of just the current session with current cmdlet.
- Informs user about **PSReadLine**.

File: reference/5.1/Microsoft.PowerShell.Core/Add-History.md [hunk 2]
```diff
@@ -21,80 +21,109 @@ Add-History [[-InputObject] <PSObject[]>] [-Passthru] [<CommonParameters>]
...
+The first command uses the `Import-Clixml` cmdlet to import a command history that was exported to
+the `History.xml` file. The pipeline operator passes the commands to the `Add-History` cmdlet, which
+adds the commands to the current session history. The **PassThru** parameter passes the objects
+representing the added commands down the pipeline.
 
-The command then uses the ForEach-Object cmdlet to apply the Invoke-History command to each of the commands in the combined history.
-The **Invoke-History** command is formatted as a script block, enclosed in braces, as required by the *Process* parameter of the **ForEach-Object** cmdlet.
+The command then uses the `ForEach-Object` cmdlet to apply the `Invoke-History` command to each of
+the commands in the combined history. The `Invoke-History` command is formatted as a script block,
+enclosed in braces, as required by the **Process** parameter of the `ForEach-Object` cmdlet.
```
Comment (chasewilson) [inline reference/5.1/Microsoft.PowerShell.Core/Add-History.md:78]:
```suggestion
enclosed in braces (`{}`), as required by the **Process** parameter of the `ForEach-Object` cmdlet.
```

File: reference/5.1/Microsoft.PowerShell.Core/Get-History.md [hunk 3]
```diff
@@ -21,80 +21,83 @@ Get-History [[-Id] <Int64[]>] [[-Count] <Int32>] [<CommonParameters>]
...
-The **Get-History** cmdlet gets the session history, that is, the list of commands entered during the current session.
+The `Get-History` cmdlet gets the session history, that is, the list of commands entered during the
+current session.
 
-PowerShell automatically maintains a history of each session.
-The number of entries in the session history is determined by the value of the $MaximumHistoryCount preference variable.
-Beginning in Windows PowerShell 3.0, the default value is 4096.
+PowerShell automatically maintains a history of each session. The number of entries in the session
+history is determined by the value of the `$MaximumHistoryCount` preference variable. Beginning in
+Windows PowerShell 3.0, the default value is **4096**. By default, history files are saved in the
```
Comment (chasewilson) [inline reference/5.1/Microsoft.PowerShell.Core/Get-History.md:29]: I think values should be wrapped in backticks?

File: reference/5.1/Microsoft.PowerShell.Core/About/about_History.md [hunk 4]
```diff
@@ -17,6 +17,25 @@ When you enter a command at the command prompt, PowerShell saves the command in
...
+The history is written to a cental file per host. That history file is
+available to all sessions and contains all past history. The history is not
+deleted when the session ends. Also, that history cannot be managed by the
+`*-History` cmdlets. For more information, see
+[about_PSReadLine](../../PSReadLine/About/about_PSReadLine.md).
+
+## Using the built-in session history
+
+The built-in history only tracks the commands used in the current session. The
+history is not available to other session and is deleted when the session ends.
```
[the remark this answers is not in the document]
Comment (sdwheeler) [reply reference/5.1/Microsoft.PowerShell.Core/About/about_History.md:37]: Fixed.

File: reference/5.1/Microsoft.PowerShell.Core/About/about_History.md [hunk 5]
```diff
@@ -17,6 +17,25 @@ When you enter a command at the command prompt, PowerShell saves the command in
 the command history. You can use the commands in the history as a record of
 your work. And, you can recall and run the commands from the command history.
 
+PowerShell has two different history providers: the built-in history and the
+history managed by the **PSReadLine** module. The histories are managed
```
[the remark this answers is not in the document]
Comment (sdwheeler) [reply reference/5.1/Microsoft.PowerShell.Core/About/about_History.md:21]: I know what you mean but the comma is not needed.

File: reference/5.1/Microsoft.PowerShell.Core/About/about_History.md [hunk 6]
```diff
@@ -17,6 +17,25 @@ When you enter a command at the command prompt, PowerShell saves the command in
...
+separately, but both histories are available in sessions where **PSReadLine**
+is loaded.
+
+## Using the PSReadLine history
+
+The PSReadLine history tracks the commands used, in all PowerShell sessions.
```
[the remark this answers is not in the document]
Comment (sdwheeler) [reply reference/5.1/Microsoft.PowerShell.Core/About/about_History.md:27]: Fixed

File: reference/5.1/Microsoft.PowerShell.Core/About/about_History.md [hunk 7]
```diff
@@ -17,6 +17,25 @@ When you enter a command at the command prompt, PowerShell saves the command in
...
+The history is written to a cental file per host. That history file is
```
[the remark this answers is not in the document]
Comment (sdwheeler) [reply reference/5.1/Microsoft.PowerShell.Core/About/about_History.md:28]: Fixed

File: reference/5.1/Microsoft.PowerShell.Core/Add-History.md (hunk 1 above)
Comment (sdwheeler) [reply reference/5.1/Microsoft.PowerShell.Core/Add-History.md:33]: Reordered

File: reference/5.1/Microsoft.PowerShell.Core/Add-History.md (hunk 2 above)
Comment (sdwheeler) [reply reference/5.1/Microsoft.PowerShell.Core/Add-History.md:78]: Added

File: reference/5.1/Microsoft.PowerShell.Core/Get-History.md (hunk 3 above)
Comment (sdwheeler) [reply reference/5.1/Microsoft.PowerShell.Core/Get-History.md]: fixed

**Pull request merged**